src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/DocumentUriExtensions.cs

```diff
@@ -15,10 +14,5 @@ public static string GetAbsoluteOrUNCPath(this DocumentUri documentUri!!)
         {
             return documentUri.ToUri().GetAbsoluteOrUNCPath();
         }
-
-        public static string GetAbsolutePath(this DocumentUri documentUri!!)
```

Comment:
Only used by our semantic tokens logic, which I changed to instead call `GetAbsoluteOrUNCPath`